_config.yml

```diff
@@ -0,0 +1,111 @@
+# Site settings
+title: Pyronear
+description: "Détection précoce de feux de forêt" # this means to ignore newlines until "baseurl:"
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "" # the base hostname & protocol for your site
+
+twitter_username: pyro_near
+facebook_username: pyronear
+linkedin_username: pyronearfr
+github_username: pyronear
+
+# Build settings
+markdown: kramdown
+
+# Traduction
+t:
+  fr:
+    buttons:
+      moreInfo: "En savoir plus"
+      contactUs: "Nous contacter"
+      submit: "Envoyer"
+    header:
+      title: "Détection précoce des départs de feux"
+      subtitle: "Pyronear est un projet open-source au service du bien commun."
```

Comment:
I will remove the dot at the end, I always find strange a dot in a title or subtitle. But it's personal

Reply:
Fixed!